src/inmanta/server/server.py

```diff
@@ -169,6 +143,7 @@ async def collect_for_slice(slice_name: str, slice: protocol.ServerSlice) -> Sli
                 FeatureStatus(slice=feature.slice, name=feature.name, value=self.feature_manager.get_value(feature))
                 for feature in self.feature_manager.get_features()
             ],
+            status=max(slices, key=lambda slice: STATUS_ORDER[slice.reported_status]).reported_status,
```

Comment:
neat. You could add the integer value to the enumeration, but this works as well .

Reply:
Something like that should work:
```py
from enum import StrEnum

class ReportedStatus(StrEnum):
    OK = "OK"
    Warning = "Warning"
    Error = "Error"

    def __gt__(self, other: "ReportedStatus"):
        order = [ReportedStatus.OK, ReportedStatus.Warning, ReportedStatus.Error]
        if self not in order or other not in order:
            raise ValueError
        return order.index(self) > order.index(other)


ok = ReportedStatus.OK
wa = ReportedStatus.Warning
er = ReportedStatus.Error


print(max([wa, er, ok]))    # Error
print(max([wa, er]))        # Error
print(max([wa, ok]))        # Warning
print(max([ok]))            # OK





```